src/components/VideoPlayerContexts/PlaybackContext.tsx

```diff
@@ -94,12 +100,23 @@ function PlaybackContextProvider({children}: ChildrenProps) {
         // This prevents the video that plays when the app opens from being interrupted when currentReportID
         // is initially empty or '-1', or when it changes from empty/'-1' to another value
         // after the report screen in the central pane is mounted on the large screen.
-        if (!currentReportID || !prevCurrentReportID || currentReportID === '-1' || prevCurrentReportID === '-1') {
+        if (!currentReportID || !prevCurrentReportID || currentReportID === '-1' || prevCurrentReportID === '-1' || currentReportID === prevCurrentReportID) {
```

Comment:
I added this since `resetVideoPlayerData` is called which clears the ref and the pending play video below won't do anything since the ref is already null. So, when we come back to the app/tab, the video isn't played.